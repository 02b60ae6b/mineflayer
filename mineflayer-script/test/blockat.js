mf.setTimeout(function() {
    mf.print("Test timed out.");
    mf.exit();
}, 5000);
mf.onSpawn(function() {
    // set timeout to wait for blocks to load...
    mf.setTimeout(function(){
        var pt = new mf.Point(0,0,0);
        for (pt.z = 58; pt.z <= 62; pt.z++) {
            for (pt.y = 15; pt.y <= 19; pt.y++) {
                for (pt.x = 103; pt.x <= 107; pt.x++) {
                    mf.print(mf.blockAt(pt).type + ",");
                }
                mf.print("\n");
            }
            mf.print("\n");
        }
        mf.exit();
    }, 3000);
});
