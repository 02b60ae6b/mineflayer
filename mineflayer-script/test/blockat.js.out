3,3,3,3,3,
12,12,12,12,12,
1,24,1,1,1,
1,24,24,1,1,
1,1,1,1,1,

3,3,3,3,3,
12,12,12,12,12,
12,12,12,24,24,
1,24,24,1,1,
1,1,1,1,1,

9,9,9,9,9,
12,12,12,12,12,
12,12,12,12,12,
1,24,24,1,1,
1,1,1,1,1,

9,9,9,9,9,
12,12,12,12,12,
12,12,12,12,12,
12,12,12,1,24,
1,1,1,1,1,

9,9,9,9,9,
9,9,9,9,9,
12,12,12,12,12,
12,12,12,12,12,
12,1,1,1,1,

